### ISSUE TITLE
Nan Again

### ISSUE THREAD

**Issue by dahangli** [davidsandberg/facenet#269]
Nan Again！
Epoch: [27][359/1000] Time 0.938 Loss 8.829 RegLoss 0.213
Epoch: [27][360/1000] Time 0.937 Loss 8.820 RegLoss 0.213
Epoch: [27][361/1000] Time 0.938 Loss 9.171 RegLoss 0.213
Epoch: [27][362/1000] Time 0.955 Loss 8.713 RegLoss 0.213
Epoch: [27][363/1000] Time 0.922 Loss 8.978 RegLoss 0.213
Epoch: [27][364/1000] Time 0.923 Loss 9.253 RegLoss 0.213
Epoch: [27][365/1000] Time 0.906 Loss nan RegLoss nan
Epoch: [27][366/1000] Time 0.916 Loss nan RegLoss nan
Epoch: [27][367/1000] Time 0.906 Loss nan RegLoss nan
Epoch: [27][368/1000] Time 0.922 Loss nan RegLoss nan
Epoch: [27][369/1000] Time 0.892 Loss nan RegLoss nan
Epoch: [27][370/1000] Time 0.875 Loss nan RegLoss nan
Epoch: [27][371/1000] Time 0.922 Loss nan RegLoss nan
Epoch: [27][372/1000] Time 0.929 Loss nan RegLoss nan
Epoch: [27][373/1000] Time 0.906 Loss nan RegLoss nan
Epoch: [27][374/1000] Time 0.988 Loss nan RegLoss nan
Epoch: [27][375/1000] Time 0.938 Loss nan RegLoss nan
Epoch: [27][376/1000] Time 0.922 Loss nan RegLoss nan
Epoch: [27][377/1000] Time 0.922 Loss nan RegLoss nan
Epoch: [27][378/1000] Time 0.906 Loss nan RegLoss nan

I edit “learning_rate_schedule_classifier_casia.txt”
But it seen to no use.

**Comment by dahangli** [comment]
Epoch: [0][10/1000]     Time 0.859      Loss 14.378     RegLoss 3.588
Epoch: [0][11/1000]     Time 0.859      Loss 14.402     RegLoss 3.611
Epoch: [0][12/1000]     Time 0.844      Loss 14.915     RegLoss 3.642
Epoch: [0][13/1000]     Time 0.875      Loss 13.983     RegLoss 3.690
Epoch: [0][14/1000]     Time 0.844      Loss nan        RegLoss nan
Epoch: [0][15/1000]     Time 0.859      Loss nan        RegLoss nan
Epoch: [0][16/1000]     Time 0.844      Loss nan        RegLoss nan
Epoch: [0][17/1000]     Time 0.875      Loss nan        RegLoss nan
Epoch: [0][18/1000]     Time 0.859      Loss nan        RegLoss nan


It's too great chance to get Nan
